event/src/args.rs

```diff
@@ -0,0 +1,170 @@
+use crate::deterministic_event::DeterministicEvent;
+use crate::event::Event;
+use crate::{DidDocument, StreamId};
+
+use anyhow::Result;
+use ceramic_core::{Cid, Jwk};
+use once_cell::sync::Lazy;
+use rand::Fill;
+use serde::Serialize;
+use std::str::FromStr;
+
+const SEP: &str = "model";
+
+// All models have this parent StreamId, while all documents have the parent stream id of the model
+// they use
+pub static PARENT_STREAM_ID: Lazy<StreamId> =
+    Lazy::new(|| StreamId::from_str("kh4q0ozorrgaq2mezktnrmdwleo1d").unwrap());
+
+pub struct EventArgs<'a> {
+    signer: &'a DidDocument,
+    controllers: Vec<&'a DidDocument>,
+    parent: &'a StreamId,
+    sep: &'a str,
+}
+
+impl<'a> EventArgs<'a> {
+    pub fn new(signer: &'a DidDocument) -> Self {
+        Self {
+            signer,
+            controllers: vec![signer],
+            parent: &PARENT_STREAM_ID,
+            sep: SEP,
+        }
+    }
+
+    pub fn new_with_parent(signer: &'a DidDocument, parent: &'a StreamId) -> Self {
+        Self {
+            signer,
+            controllers: vec![signer],
+            parent,
+            sep: SEP,
+        }
+    }
+
+    pub fn parent(&self) -> &StreamId {
+        self.parent
+    }
+
+    pub async fn init(&self) -> Result<DeterministicEvent> {
+        let cmt = UnsignedEvent::<()>::init(self)?;
```

Comment:
nit, but the name `cmt` was confusing at first. Then I realized it mean `commit`. Could you rename the uses of `cmt` to something more clear, maybe `evt` or `event`?